cmd/api.go

```diff
@@ -98,6 +100,12 @@ var apiCmd = &cobra.Command{
 		}
 		log.Infof("Connected to Redis at %s", redisURI)
 
+		// Connect to Memcached if it exists
+		mem, err := datastore.NewMemcached(networkInfo.Name, memcachedEndpoints...)
```

Comment:
does this not error if no memcache endpoints are set?

Reply:
I tested it with no memcached endpoints set and it doesn't error - there's a unit test for datastore that confirms the behavior

Reply:
discussed offline, Please feel free to un-resolve if missing anything